electron-app/src/server/websites/mastodon/mastodon.service.ts

```diff
@@ -220,6 +220,19 @@ export class Mastodon extends Website {
         };
       }
 
+      // Tags only should be posted on public entries - they can not be searched on other types
+      if (options.visibility == 'public') {
+        // Update the post content with the Tags if any are specified - for Mastodon, we need to append 
+        // these onto the post, *IF* there is character count available.
+        data.tags.forEach(tag => {
+          let remain = maxChars - form.status.length;
+          if (remain > (tag.length)) {
+            form.status += ` ${tag}`
```

Comment:
q: Are tags really just plaintext on Mastodon? No hashes or otherwise? This seems like it's just jamming in keywords at the end of the status.

nit: Missing a newline insertion, so these would just trail off the last char in the status I think.

suggestion: It might be worth adding an additional toggle to append tags.

Reply:
Yup, they (currently!) just get appended - or put anywhere - in the post text. Bit like Twitter really.  
![image](https://user-images.githubusercontent.com/6953049/229364254-1f657f27-ea1c-44ab-b9b3-c9425b210069.png)

Was thinking about the newline addition myself after experimenting - will update the branch shortly!

At present this is using the existing toggles - are you thinking there should be an additional one?

![image](https://user-images.githubusercontent.com/6953049/229364506-d98a3435-1509-4419-8ef1-b1c61f596691.png)

Reply:
I am somewhat on the fence when it comes to appending tags to the status of a message is all. I guess worst case someone can override tags and keep it empty.

In your example post provided, I see that your tags are #<tag>

I don't see the hash being appended, is what I was meaning

Reply:
Ahh yeah! I did originally have a post where I was adding # to the tag as well, as its needed (like on Twitter) however ended up with a duplicate # being posted. Seemed the tag value is inserting the # automatically at this point? Didn't do much digging to be fair as the output was correct with the has being prepended!

Reply:
Sorry for the slow replies, I've been unusually busy.

I am surprised to hear about the # being there automatically since I don't think that is the default behavior of the app. I can try testing this out, but it should have only been applying the hash if the tag itself was created with a hash already.